desafio-6/desafio-6.css

```diff
@@ -0,0 +1,53 @@
+body{
+  font-family: arial, sans-serif;
+  background: #8FBC8F;
+  margin: 0;
+}
+p{
+  padding: 20px;
+  text-align: justify;
+}
+
+.container{
+  height: 60px;
+  background-position: right center;
+  background-size: cover;
+  border-bottom: 1px solid #ffffff;
+}
+
+.container-info{
+  text-align: center;
+  color: white;
+  margin-top: 30px;
+}
+
+.container-info h1{
+  text-transform: uppercase;
+  font-size: 30px;
+  text-shadow: 3px 3px 2px #333;
+}
+
+.container nav ul{
+  text-align: right;
+  margin: 20px 50px 0 0;
+}
+
+.container nav li{
+  display: inline-block;
+  list-style: none;
+  margin-left: 10px;
```

Comment:
After applying display flex, you can remove this margin and use the property `gap` in the tag that the display was added to

Reply:
changed rai, thank you very much <3